Translate Shadow Succubus (#478)
Giratena/Succubus-Duel-English-Translation: CSV/Language/English/Card/10072.txt

default
淫魔化シャドー,Shadow Succubus
EU72_paizuriAnime,
EU72_bustShakeAnime,
EU72_bustShowShakeAnime,

> エルルが使用していたカード「シャドー」がサキュバス化した姿。元々女の子だったが、サキュバス化によって魅力的な体を手に入れ、色気がそなわった。影の姿をしていたときは誰も女の子扱いをしてくれないのを気にしていたので、サキュバス化したことを小躍りして喜んでいる。\n胸が豊かに育ち、その感触は極上の一言。エルルやキャロルの動きを見ていたので、パイズリや手コキなども上手にこなす。うかつに挟まれてしまえば、蕩けるような快楽で動けなくなり、そのまま深い影の中にひきずり込まれてしまうだろう。,The succubus form of the "Shadow" card Elulu once used. While she has always been female; her transformation has granted her an attractive body and an aura of sex appeal she once lacked. As a shadow; she was often bothered by the way no one seemed to treat her like a girl; leaving her overjoyed to have become a succubus. Her breasts have grown abundantly; and their texture is superb. Having watched Elulu and Carol's techniques for a long time; she can skillfully perform paizuri and handjobs. If someone were to be carelessly caught between her breasts; they'd be left unable to move from the melting pleasure and dragged into the depths of her shadows.

Reviewer: I thought it was "Eruru" instead of "Elulu"

Also on line 113 and 169

Author: So Gira and I discussed it and we're actually going with Elulu isntead since 'Lulu' is an actual english name, and it just kinda sounds nicer in english.

Reviewer: Sounds good 👌

> エルルが使用していたカード「シャドー」がサキュバス化した姿。元々女の子だったが、サキュバス化によって魅力的な体を手に入れ、色気がそなわった。影の姿をしていたときは誰も女の子扱いをしてくれないのを気にしていたので、サキュバス化したことを小躍りして喜んでいる。\n胸が豊かに育ち、その感触は極上の一言。エルルやキャロルの動きを見ていたので、パイズリや手コキなども上手にこなす。うかつに挟まれてしまえば、蕩けるような快楽で動けなくなり、そのまま深い影の中にひきずり込まれてしまうだろう。,The succubus form of the "Shadow" card Elulu once used. While she has always been female; her transformation has granted her an attractive body and an aura of sex appeal she once lacked. As a shadow; she was often bothered by the way no one seemed to treat her like a girl; leaving her overjoyed to have become a succubus. Her breasts have grown abundantly; and their texture is superb. Having watched Elulu and Carol's techniques for a long time; she can skillfully perform paizuri and handjobs. If someone were to be carelessly caught between her breasts; they'd be left unable to move from the melting pleasure and dragged into the depths of her shadows.

Reviewer 2: Hmm I wonder if this could have an improved flow. Here are my suggestions!

```suggestion
エルルが使用していたカード「シャドー」がサキュバス化した姿。元々女の子だったが、サキュバス化によって魅力的な体を手に入れ、色気がそなわった。影の姿をしていたときは誰も女の子扱いをしてくれないのを気にしていたので、サキュバス化したことを小躍りして喜んでいる。\n胸が豊かに育ち、その感触は極上の一言。エルルやキャロルの動きを見ていたので、パイズリや手コキなども上手にこなす。うかつに挟まれてしまえば、蕩けるような快楽で動けなくなり、そのまま深い影の中にひきずり込まれてしまうだろう。,A succubus-ified form of Elulu's 'Shadow' monster card. While she has always been female; her transformation bestowed her an incredibly attractive body—a body designed for seduction that her shadowy form had never possessed. Originally as a shadow; she was often bothered by the way no one seemed to treat her like a girl; leaving her overjoyed to have become a succubus. Her breasts have grown abundantly; and their texture is superb. Having watched Elulu and Carol's techniques for a long time; she can skillfully perform paizuri and handjobs. A man caught between those soft; pillowy breasts would be paralyzed by pleasure—her flesh swaying and jiggling until resistance crumbled; leaving only a mindless thrall to be swallowed by her shadows.
```

Author: Two things here:

1. I really dont like the term 'succubus-ifed.' It feels clunky.
2. The 'a body designed for seduction that her shadowy form had never possessed' part also feels really weird, but not in a way I can put my finger on.

Reviewer 2: Succubus-ification is the proper term to my knowledge and one I've seen before.

I don't see anything wrong with point 2! One of the things I look for are weird phrases, and "an aura of sex appeal" isn't one that is commonly used, nor does it fit the sexy tone.

周りのことも見えないし、周りからも見られない❤\n他の人たちのことなんて忘れて、\n楽しい暗闇時間を過ごそうね～❤えいっ、影の中にずぷずぷ～❤,Lost in the shadows; you can't see anyone; and nobody can see you.❤ Go ahead and just forget all about everyone else. Let's have some fun in the darkness together.❤ Come here; slip into the shadows with me~❤
手でシコシコしてあげる❤\n長ーい袖の中におちんちん入れてぇ、\nほら、シコシコ❤シコシコ❤,I'll rub your penis for you.❤ I'll put your dick inside my long sleeve; and... Stroke-stroke~❤ Stroke-stroke~❤
袖の中は真っ暗闇❤外からじゃ何されているか見えないね❤\n予想できないタイミングで、おちんちんに指を絡みつけてぇ、\n竿をシコシコ❤先っぽと玉を、さわさわ❤カリカリ❤,It's so dark in my sleeve; huh?❤ You can't see what I'm doing from outside at all; can you?❤ Without warning; I'll wrap my fingers around your cock and stroke your shaft.❤ Or maybe I'll tease the tip or balls.❤ Tickle tickle~❤
あは、おちんちんがビクンって震えて、我慢汁が出てきたよ❤\n暗くて暗くてこわいよー❤って泣いてるみたい❤\nほらぁ、もっとなでなでしてあげるから、泣かないで❤きゃはは❤,Aha; your cock is twitching and starting to leak!❤ It's almost like it's crying; like it's scared of the dark~❤ It's okay. I'll pet it more; so don't cry; okay?❤ Kyahaha~❤
えー、ボクの前で恥ずかしそうにしちゃってどうしたの？\nなになに？ボクに気持ちよくしてほしい？,Huh? Why are you acting all shy around me? What's that? You want me to make you feel good?

> きゃはは❤信じられない❤\nみんなの命がかかってるって時でも、気持ちいいの優先しちゃうんだ❤\nうわー、キミ、最低だねー❤おちんちんそんなに我慢できなかった？,Kyahaha~❤ Unbelievable!❤ Even when everyone's lives are at stake; feeling good is all that matters to you.❤ Wow; you're the worst~❤ Couldn't you hold your cock back; even just a little?

Reviewer 2: ```suggestion
きゃはは❤信じられない❤\nみんなの命がかかってるって時でも、気持ちいいの優先しちゃうんだ❤\nうわー、キミ、最低だねー❤おちんちんそんなに我慢できなかった？,Kyahaha~❤ Unbelievable!❤ Even when everyone's lives are at stake; feeling good is all that matters to you.❤ Wow; you're the worst~❤ You really couldn't control your dick; even just a little?
```

> ボクのおっぱいが大きすぎるのがいけなかったのかなー❤\nいひひ❤いいよ、気持ちよく吸い殺してあげる❤,Maybe it was just bad luck for you that my breasts got this big.❤ Ihihi~❤ Fine; I'll just go ahead and pleasure you to death!❤

Reviewer 2: Maybe a different laugh than "Ihihi"

Author: Still waiting on that suggestion.

Reviewer 2: What about "Kehehe!"?

Author: That's a little too harsh for a small cutesy charcter

Reviewer 2: I think it's perfect for her archetype actually. If you look at the art she's got a devlish smile. Definitely not small nor cute.
```suggestion

ボクのおっぱいが大きすぎるのがいけなかったのかなー❤\nいひひ❤いいよ、気持ちよく吸い殺してあげる❤,Maybe it was just bad luck for you that my breasts got this big.❤ Kehehe~❤ Fine; I'll just go ahead and pleasure you to death!❤
```

あは、ボクに抱きしめられながらびくびくしてる～❤\nボクのおっぱいが気持ちよすぎて、痙攣しながら射精しちゃったね❤\nザーコ❤,Aha; my big hug's making you tremble so much~❤ My breasts felt so good your whole body convulsed when you came; didn't it?❤ Loser~❤
ううっ…❤我慢できない…❤,Ugh...❤ I-I can't hold back...❤
にひひ❤戦いの途中なのにおちんちんシコシコしてる❤\nボクのかわいい姿を見てたらぁ、\nもうバトルとかどうでもよくなっちゃった？,Nihihi~❤ You're actually stroking your cock in the middle of battle!❤ After seeing how cute I am now; you don't care about the battle at all anymore; do you?
ボク、元はただのちんちくりんなモンスターだったんだよ？\nサキュバスになって女の子らしくなった瞬間、\nそんな節操なく扱いちゃってさー❤,Before this I was just a tiny little monster; you know. But the moment I became a succubus and looked more feminine; you couldn't help but touch yourself so shamelessly~❤
例えばこのおっぱい❤服の中がパンパンになるぐらい大きくなって、\nこの隙間のところから溢れだしそうなんだ❤\nみっちみちに詰まっててエロいよねー❤,Look at my chest; for example.❤ My boobs've gotten so big that my clothes are almost bursting at the seams; and they look like they might spill out of this little gap here.❤ They're packed in so tight and sexy~❤

> キミのおちんちんを、この隙間に入れてるところ想像して？\nボクのおっぱいをむにむに～ってかき分けて、\nおちんちんを中まで突き入れてぇ・・・❤,Just imagine sliding your cock into this gap. Pushing my breasts apart with a big squish and thrusting your cock deep inside...❤

Reviewer: Preference here, but I like keeping the question on the first sentence:

> **Can you** imagine sliding your cock into this gap? Pushing my breasts apart with a big squish and thrusting your cock deep inside...❤

Author: Hmm okay. What about. "Your cock could slide so nicely into this gap. Can you just imagine/picture it?"

Reviewer: That looks good to me! Though I'd drop 'just' and use "Can you picture it?"

Author: Went with 'Can't you just picture it' instead. What do you think? I feel like it has a better teasing ring, you know?

Reviewer: Works for me!

> 入ったところで、乳揺れサービス❤\nあんあんってわざとらしい声をあげながら\nおっぱいを揺らしてあげるの❤,Once it was in; I'd give it a nice booby-bouncing service.❤ I'd make exaggerated moaning sounds while shaking my chest; just for you~❤

Reviewer 2: Hmm the tense for this line feels a little off: 
```suggestion
入ったところで、乳揺れサービス❤\nあんあんってわざとらしい声をあげながら\nおっぱいを揺らしてあげるの❤,Once it's in; I'll  give it a booby-bouncing service.❤ I'll even make exaggerated moaning sounds while shaking my chest; just for you~❤
```

Author: So I'm okay with the tense change, but the other changes less so. Since 'cock' was mentioned just one line ago, I dont feel like it needs to be restated, I don't really like the 'you're in,' part since it feels like it's lessening the focus of the comments she's making, and while I could be convinced, I personally really like the light hearted feel of 'booby bouncing service'

Reviewer 2: Boobie bouncing service is OK to use, but not with "nice" in front of it. It's a little overkill.

"Once it's in" is ok to use as well.

ねえねえキミは分かってるのかな❤\nボクで気持ちよくなってびゅーって出しちゃったら、\nエナジー吸われて死んじゃうかもしれないんだよ？,Hey; be careful~❤ If you feel too good and end up cumming your brains out; your energy might get drained and you could die. You know that; right?
それに、倒さなきゃいけないボクを回復させちゃう❤\n他の仲間にとったら、最低の裏切り行為だよね❤\nだから、ホントはぜったいおちんちんシコシコ止めなきゃだめ❤,Plus; you'll end up healing me instead of defeating me like you're supposed to.❤ You'd end up failing your allies. It'd be the ultimate act of betrayal!❤ So you really; really have to stop stroking your cock; got it?❤
ふふ、分かっててもやめられないんだ❤\nしこしこ、しこしこ♪目をとろーんとさせながら動かしてる❤,Fufu; even knowing all that; you can't stop; can you?❤ Stroke-stroke; stroke-stroke~♪ You're rubbing it so desperately; your eyes all glazed over~❤
大きなおっぱいゆさゆさってされたら、\nキミの弱い理性じゃ我慢できないんだよね❤おちんちん\nどんどん気持ちよくなってオナニーで脳みそとろとろになっちゃう❤,When breasts this big get jiggled in front of you; a will as weak as yours just can't resist.❤ Your cock keeps feeling better and better until every stroke turns your brain to mush~❤

> ほらほらほらぁ、キミに仲間を裏切らせた原因の\nボクの最強おっぱいだよー❤\nにひひ、しこれしこれ❤さっさと死んじゃえ❤,That's it; look; look! These are the irresistible breasts that made you betray your allies~❤ Nihihi; stroke it; stroke it~❤ Hurry up and die already!❤

Reviewer 2: Hmm "These are the irresistible breasts that made you betray your allies~" sounds a little flat

Author: Do you have any suggestions on what's flat about it? Would you rather 'hypnotic breasts' or something like that?

Reviewer 2: Maybe something along the lines of: "these boobs are practically hypnotic- you'd do anything for them, even betray your allies~"

Author: That doesn't really fit seeing as how at this point he's already betrayed them

Reviewer 2: Hmm not sure what you mean. The sentence still makes sense? She's stating it as a fact, which is perfect for the use of よ here.

射精したら死ぬかもしれないんだもん。\n一秒でも長く楽しみたいよね❤\nでも、ダメ❤さっさと終わらせちゃう❤,You might die if you cum right now; you know. Don't you want hang on and enjoy this more; even for just one more second?❤ Well too bad!❤ I'm gonna finish you off here and now~❤
ほらバーカ❤\n思い切りおっぱい揺らしてあげるから、\nさっさとイってエナジー全部おっぱいに食べられちゃえ❤,Take this; you idiot!❤ I'm going to bounce my boobs as hard as I can; so hurry up and cum; and let my tits devour all your yummy energy!❤
この谷間にぜんぶ搾り取られてぇ❤\nさっさと死んでいいよ❤マーゾ❤,Let my cleavage milk every last drop out of you!❤ Hurry up and die already!❤ You masochist!❤
はひぃっ・・・❤,Hah; aaahh...❤
にひひ、どんだけ出してるの？このヘンタイ❤\nボクのおっぱいで、エナジーを吸収っと❤,Nihihi; why are you cumming so much? You pervert!❤ My boobs are sucking your life away; you know!❤
あは、おいしい～❤\nそれにしてもサキュバスになったらホント簡単に勝てちゃうんだね❤\n男ってちょろい❤くすくす❤,Aha; so yummy~❤ Now that I'm a succubus; winning's gotten so much easier!❤ Boys like you are so stupid~❤ *Giggle*❤
にひひ❤戦いの途中なのにおちんちんシコシコしてる❤\nボクのかわいい姿を見てたらぁ、\nキミもバトルとかどうでもよくなっちゃったの？,Nihihi~❤ Even though we're in the middle of battle; you actually started stroking your cock!❤ The moment you saw my cute new look you just stopped caring about the battle too; didn't you?

> やぁん、ボク魅力的すぎて困っちゃう❤\nほら、キミにも近くでおっぱいを見せてあげる❤,Too bad; guess I'm just too sexy for my own good!❤ Look; I'll show you my boobs up close and personal; just like the last guy!❤

Reviewer: I want to capture the cutesy/coy feeling of やぁん here, but I don't have a great suggestion.
Maybe use a tilde punctuation?

Author: Maybe 'oopsy'? Would that be too much?

Reviewer: I like it!

> 服の中にみっちり詰まったサキュバスおっぱいだよ❤\nキミの頭ぐらいある大きなおっぱい❤\n谷間の隙間からこぼれ出しそうになっててエロいよねー❤,My new succubus tits are crammed so tightly inside my clothes!❤ My big boobs are the size of your head!❤ They're about to spill out of the hole in my clothes; they're so big. Pretty sexy; huh?❤

Reviewer: Slight rewording suggestion?

> My succubus tits are crammed so tightly inside my clothes!❤ **These** big boobs are **as large as** your head~❤ They're **practically spilling** out of the hole in my clothes. Pretty sexy; **right?❤**

Author: Hmmm for some reason 'large' doesnt feel right to me. Maybe we try to fit 'big' where 'large' is and find another word for 'big'? Like 'These huge boobs are as big as your head'

Reviewer: Yeah, that revision works better -

I was also trying to think of a way to revise "hole in my clothes" too... but I don't think saying "cleavage gap" is intuitively understood even with the picture.

Is it too much of a stretch to use "They're practically bursting out of this revealing top."?

Author: That MIGHT be a stretch, yeah. Would making it 'gap' or 'slit' instead of 'hole' help?

(For the record, I think she's wearing a dress so 'top' doesnt really work because it's a one piece)

Reviewer: I feel like 'gap' works well 👍

> 服の中にみっちり詰まったサキュバスおっぱいだよ❤\nキミの頭ぐらいある大きなおっぱい❤\n谷間の隙間からこぼれ出しそうになっててエロいよねー❤,My new succubus tits are crammed so tightly inside my clothes!❤ My big boobs are the size of your head!❤ They're about to spill out of the hole in my clothes; they're so big. Pretty sexy; huh?❤

Reviewer 2: Hmm the last sentence feels kind of strange to me. Is there a way to write it that causes it to flow more naturally?

Author: For some reason my change from Wiz's review didn't take. It now says 'They're practically spilling out of the gap in my clothes; they're so big. Pretty sexy; right?❤'

男がいくら頑張ってボクを追い詰めても、\nこのえろーい体で誘惑してあげれば関係ないみたい❤\n動きが止まって天国パイズリであっという間に形勢逆転♪,No matter how hard men try to corner me; as long as I can seduce them with my sexy new body; it doesn't matter!❤ They stop moving; letting me turn the tables in an instant with a heavenly titfuck~♪
にひひ❤サキュバスのパイズリ気持ちいいでしょ～❤\n頭アヘアへにしちゃう成分とか出てるんだよ～、たぶん。\nだから、ほらぁ、早くズリズリってしてあげれば・・・。,Nihihi~❤ A boobjob from a succubus feels so good; huh?❤ There's probably some pheromone I'm letting out that makes your head all silly. Because of that; look; if I stroke you quickly like this...
はひっ、だめっ、だめぇっ❤\nきもちよくてっ、そんなにこすったら、あああっ❤❤,Hah; no; noo!❤ It feels too good! I-if you rub it like that... Aaaah!❤❤
ぷぷっ、イっちゃっていいよー❤\nボクのボロボロになった体を、キミの命で回復させてぇ❤\nおねがーい❤エナジーちょうだいちょうだい❤,Pfft; if you wanna cum; just do it!❤ Use your life to heal my battered body~❤ Please?❤ Give me your energy; give it all to me!❤
このおっぱいにめためたのぐちゅぐちゅに犯されて、\nほらぁ、谷間の中に出しちゃえ❤\n命を出して、干からびちゃえ❤くすくすくす♪,Get completely dominated by these tits until you're a total mess! Come; on; cum inside my cleavage!❤ Shoot your life out and shrivel up~❤ *Giggle*♪
あひぃっ・・❤❤,Ahhh...❤❤

> きゃはは❤すごーい超どくどく出てる❤\nおっぱいがエナジーに満たされて、あつぅい❤,Kyahaha~❤ Amazing; you're cock's throbbing and spurting so much!❤ My boobs are filling with energy; and it's making them so hot!❤

Reviewer 2: Hmm maybe something like this?
```suggestion
きゃはは❤すごーい超どくどく出てる❤\nおっぱいがエナジーに満たされて、あつぅい❤,Kyahaha~❤ Amazing; you're cock's throbbing and spurting so much!❤ My boobs are filling with energy! Ooh, they're getting all hot~❤
```

Author: I fell like that's so underwhelming compared to what I currently have, you know?

Reviewer 2: But the phrase you have originally of "making them so hot!" is quite unusual, no?
I don't see anything wrong with the new suggestion.

Author: Why is it unusual? I've heard cum described as 'hot' all the time, and seen it like this a time or two. And 'i feel them getting warmer' is just so passive.

Reviewer 2: You're probably thinking of when "hot" is used as a feeling, but the current sentence is quite ambiguous. It might even sound like the act she is doing is hot.

> きゃはは❤すごーい超どくどく出てる❤\nおっぱいがエナジーに満たされて、あつぅい❤,Kyahaha~❤ Amazing; you're cock's throbbing and spurting so much!❤ My boobs are filling with energy! Ooh, they're getting all hot~❤

Author: ```suggestion
きゃはは❤すごーい超どくどく出てる❤\nおっぱいがエナジーに満たされて、あつぅい❤,Kyahaha~❤ Amazing; you're cock's throbbing and spurting so much!❤ My boobs are filling with energy! Ooh; they're getting all hot~❤
```

ボクのおっぱい気持ちよかった？\nくす、ボクみたいな下級サキュバスにいいようにされちゃって、\n男の子って弱い❤エナジーありがたくもらって回復しちゃうね♪,Did my chest feel good? *Giggle* You got completely dominated by a low-level succubus like me. Boys are so weak!❤ I'll happily take your energy and recover~♪
あう、またやられちゃいそう。\nさすがにBランクマスターは強いねー❤\nで、も・・・❤,Aww; I'm about to get beaten again. B-rank masters sure are strong~❤ But...❤
あう、またやられちゃいそう。\nさすがにBランクマスターのモンスターは強いねー❤\nで、も・・・❤,Aww; I'm about to get beaten again. Monsters of a B-rank master sure are strong~❤ But...❤
あうっ、うぐぅっ・・・❤,Ahh; ughh...❤

> あは、おっぱい見せられた瞬間、\n股間がずくんってうずいて、もうムラムラが抑えられないでしょ？\nサキュバスの誘惑魔法だよー❤効果はばつぐんみたい❤,Aha; the moment you saw my boobs; your crotch started throbbing; and you just couldn't suppress your arousal anymore; could you? That's a succubus's seduction magic!❤ The effect's really something else~❤

Reviewer 2: your crotch started throbbing is a bit of a weird phrase.

```suggestion
あは、おっぱい見せられた瞬間、\n股間がずくんってうずいて、もうムラムラが抑えられないでしょ？\nサキュバスの誘惑魔法だよー❤効果はばつぐんみたい❤,Aha; the moment you saw my boobs; I could see your crotch starting to throb. You just can't suppress your arousal anymore; can you? That's a succubus's seduction magic!❤ The effect's really something else~❤
```

Author: I feel like this is a downgrade honestly. It's more wordy and adds unneeded context with the 'I could see,' which doesnt even make sense because it's currently happening  so she's CURRENTLY seeing it, you know?

Reviewer 2: Hmm I'm not really sure I agree there. The point here is that she's telling him she noticed. Her stating it as a fact has a lot less impact since it's no longer as personal, if that makes sense.

It's little stylistic choices like these that can go a long way.

Author: If you read it with the previous lines, it just doesn't make sense. She took an action to cause this to happen. It's not like she just noticed, she did it on purpose.

Reviewer 2: Hmm, ok let's resolve this. I think the issue is minor enough to leave as is.

> ボクのおっぱいからはぁ、ちょっと怪しい魔力が出ていて、\n男の子の体にびびび～って作用しちゃうの❤\n特におっぱいが好きな子にはとっても効くみたい❤,A suspicious magical power is coming out of my boobs; inflicting boys' bodies with a buzzing sensation!❤ It seems especially effective on boys who love breasts~❤

Reviewer 2: "suspicious magical power" is a bit flat

Author: 'My boobs emit a bewitching magic now;'

Reviewer 2: Hmmmm I'm not really sure that works either.

I think the whole first sentence: "A suspicious magical power is coming out of my boobs; inflicting boys' bodies with a buzzing sensation!❤" kinda falls flat because it's telling rather than showing. I think translating directly likely lead to this point. 

Have another go, and if you can't work it out I'll see if I can provide some suggestions ^^

Author: It's going to be telling, because she's literally explaining her powers.

それじゃ、ばんざーいして降参しちゃおうね❤\nこのおっぱいでキミのこと食べてあげる❤,Well then; put your hands up and surrender; okay?❤ If you do; I'll devour you with my tits~❤

> 戦ってギンギンになっちゃったそのおちんちんを、\nパクッて挟んで、ぐちゅぐちゅにすりつぶしちゃうんだよ❤\nにひひ、気持ちよーく天国に送ってあげるね❤,Since your cock got all rock hard during our fight; I'll trap it between my tits and squeeze it into mush!❤ Nihihi; I'll make you feel so good you'll swear you're in heaven~❤

Reviewer 2: "got all rock hard" is a bit flat

Author: In what way.

Reviewer 2: The timbre of the sentence flow is weird. 

Remove the "all" and see how much better it flows?
**Since your cock got rock hard during our fight;** I'll trap it between my tits and squeeze it into mush!❤

Try that ^^

Author: I mean I guess it's fine, but it removes some of the light-hearted teasing aspect of it.

Reviewer 2: I gave this a re-read. I think it's fine as is (or at least trivial to change) so I'll just resolve this I think.

ボクのおっぱい柔らかいけど、大きさが大きさだもんね❤\n両側からむにゅうって押しつぶされて、\nおちんちん気持ちよさの洪水に溺れちゃう❤,My breasts might be soft; but they're just so big; huh?❤ Your cock's getting smushed from both sides with a nice; squishy sensation. I bet it's drowning in a flood of pleasure!❤
挟んでぱんぱんって少し動かしてあげれば、\n顔はとろとろで、喘ぎ声もたっぷり漏らして、\nくす❤キミ強いのにねー、それなのにおっぱいに勝てないんだねー❤,If I squeeze them together and move even just a little; your face goes all slack and you let out plenty of sad little moans *Giggle*❤ You're so strong; but nobody beats my boobs~❤
ボクみたいな下級サキュバスにみんないいようにされちゃってさ、\n男の子って弱いんだねー❤\nほらぁキミが出しちゃってせいで、またボクが全回復しちゃったよ❤,Everyone keeps letting themselves get totally dominated by a low-level succubus like me. Boys are so weak~❤ Look; I've fully recovered again; and all because you couldn't keep yourself from cumming!❤
お、まだ生きてるの？一回出しただけで死ななくてよかったねー❤\nオナニーしこしこで戦闘不能❤\nおっぱいドロドロにできて満足した？ザーコ❤,Oh? You're still alive? That's good. I'm kinda glad you didn't die from cumming just once~❤ Especially since you actually lost the battle from masturbating!❤ Did getting my boobs all sticky make you feel better? Weakling!❤

> ボクの逆転勝利～♪\nサキュバスおっぱいでズリズリされて最高だったでしょ？きひひ❤,A turnaround victory!♪ Getting stroked by a succubus's tits was the best; wasn't it? Kihihi~❤

Reviewer: 'A comeback victory' might be a more common phrase - optional change, since turnaround works fine

にひひ❤結局おっぱいから逃げられなくて射精しちゃったね❤\nぱふぱふしてあげただけで楽勝勝利～❤\nボクのおっぱい強すぎじゃない？,Nihihi~❤ You just couldn't escape my tits and ended up cumming!❤ An easy win; and all from just giving you some titty action~❤ These new boobs are way too strong!
にひ❤袖の中でびゅくびゅくびゅく❤\nみんなに見えないのをいいことに、\nすっごく気持ちよさそうな射精しちゃったね❤,Nihi~❤ I can feel something spurting and gushing inside my sleeve!❤ You took total advantage of no one being able to see and had a real nice orgasm; didn't you?❤
あは、腰砕けになってて弱っちい♪\nそのまま倒れたらみんなに射精したのがバレちゃうよ？\n取り繕うのをがんばれー❤あはは♪,Aha; your hips gave out! You're so weak~♪ If you collapse like that; everyone will know you came; won't they? You need to try harder to hide it.❤ Ahaha~♪
{自分}に敗れた僕は、薄暗い地下通路でエナジーを失い、地面に膝をついていた。,After being defeated by {自分} and losing my energy; I was left kneeling on the ground in the dim underground passage.
動けなくなった僕を横目にエルルは他のマスターたちのところに向かっていく。下級のマスターたちではきっと歯が立たない。このままではみんなやられてしまう・・・。,Elulu headed toward the other masters; paying me little mind as I lay on the ground; unable to move. The lower-level masters wouldn't stand a chance. At this rate; everyone will be defeated...

> いひひ❤ボクの手コキは気持ちよかったかな～❤\nサキュバスには負けないってタンカを切っていたのに、\nカッコ悪く負けちゃったね～❤,Ihihi~❤ Did my handjob feel good?❤ You were bragging about how you'd win against a succubus for sure; but you ended up losing so pathetically~❤

Reviewer 2: Ihihi again

ん？あっちのバトルが気になるの？\nキミが負けちゃったせいで、仲間が大ピンチだもんね❤,Hmm? Are you worried about that battle over there? Your teammates are in some serious trouble; and it's all because you lost~❤
いひひ❤ボクのおっぱいは気持ちよかったかな～❤\nサキュバスには負けないって啖呵を切っていたのに、\nカッコ悪く負けちゃったね～❤,Ihihi~❤ Did my breasts feel good?❤ You were bragging about how you'd win against a succubus for sure; but you ended up losing so pathetically~❤
あっちのバトルが気になるの？\nキミが負けちゃったせいで、仲間が大ピンチだもんね❤\n責任感じちゃう～？きひひ❤,Are you worried about that battle over there? Your teammates are in some serious trouble; and it's all because you lost~❤ Are you feeling a little guilty? Kihihi~❤
くっ、ううっ・・・。,Gah; uugh...
そうだ、おもしろいこと思いついちゃった♪,Oh; I just thought of something fun!♪
ねーねー、キミのカードを使ってさ、\nあっちのマスターたちを攻撃してよ❤,Hey hey; use your cards to attack those masters over there~❤
なっ・・・！,What...!?
キミに裏切られたらあいつらどんな顔をするのかな？ボク見たいなー♪\nねー、いいでしょ？死ぬ気でやれば、\nあと数枚ぐらいは召喚できるよねー❤,What do you think their faces will look like when they see you betrayed them? I want to see~♪ Come on. It's fine; right? If you give it your all; I'm sure you can still summon a few more cards; can't you?❤
くっ、そんなことするはず・・・。,Gah; I'd never do something like that...

> 僕は力の入らない体でサキュバスシャドーをにらみつける。強い怒りによって快感で麻痺していた脳が理性を取り戻すのを感じる。そうだ、僕は死んでも気絶しても構わない、無理にでもカードを召喚してせめて一太刀を・・・。,With my body drained of its strength; I glared at the Succubus Shadow. Despite being numbed by pleasure; I could feel my brain regaining its reason from the rage I was feeling. That's right. Even if I die or pass out; I don't care—I'll force myself to summon every card I can and at least land one more strike...

Reviewer 2: ```suggestion
僕は力の入らない体でサキュバスシャドーをにらみつける。強い怒りによって快感で麻痺していた脳が理性を取り戻すのを感じる。そうだ、僕は死んでも気絶しても構わない、無理にでもカードを召喚してせめて一太刀を・・・。,With my body drained of its strength; I glared at the Shadow Succubus. Despite being numbed by pleasure; I could feel my brain regaining its reason from the rage I was feeling. That's right. Even if I die or pass out; I don't care—I'll force myself to summon every card I can and at least land one more strike...
```

ええ？ほんとにだめ？\nボクがおっぱいを揺らしながら頼んでも？,Eh? Really? Even if I bounce my boobs while asking?
うりうり～❤キミが負けたおっぱいだぞ～❤\nボクのためにカードを召喚してぇ❤\nあっちのやつらをやっつけてぇ～❤,Come on; look~❤ Stare at the tits you just lost to~❤ Summon cards for me like a good boy.❤ Attack those guys over there~❤

> あうっ、あううっ❤やめてっ、揺らさないでっ❤\nううっ、サキュバスに、サキュバスなんかにっ・・・❤❤,Agh; ahh...❤ Stop it! Don't bounce them like that.❤ Ugh; losing like this... to something like a succubus...❤❤

Reviewer 2: Hmm maybe something liket this would match the vibe better?
```suggestion
あうっ、あううっ❤やめてっ、揺らさないでっ❤\nううっ、サキュバスに、サキュバスなんかにっ・・・❤❤,Agh; ahh...❤ Stop it! Don't bounce them like that.❤ Ugh; losing like this... to... to some random succubus...❤❤
```

Author: I dont know that 'random' is right. I feel like the message is that losing is bad enough, but he knows losing to a succubus -any succubus- makes it even worse.

Author: how about 'and to a succubus...' or something like that.

Reviewer 2: Hmm what about:
Ugh; losing like this... to... to **some succubus**...❤❤

That should be better ^^

Author: Why the focus on 'some' instead of 'a'? It seems overly dismissive to me

Reviewer 2: "some succubus" gives the nuance of the MC being stubborn, which contrasts with his helplesslness.

> ぷぷっ、にらんでた顔が一瞬でとろとろになっておもしろ❤\nおっぱい揺らされただけで、ボクのおねがい聞いちゃいそうだねー❤,Pfft; your glare melted in an instant. It's so funny!❤ All I did was bounce my boobs; you look like you're ready to do anything I tell you to~❤

Reviewer 2: ```suggestion
ぷぷっ、にらんでた顔が一瞬でとろとろになっておもしろ❤\nおっぱい揺らされただけで、ボクのおねがい聞いちゃいそうだねー❤,Pfft; your glare melted in an instant. It's so funny!❤ All I did was bounce my boobs; and now you look like you're ready to do anything I tell you to~❤
```

> でも、ボクってばやさしいから、\n揺らすだけじゃなくて、\nキミがやる気になれるように「おまけ」してあげる❤\,But since I'm nice; I won't just bounce them. Instead; I'll give you a little "bonus" to get you extra motivated~❤

Reviewer 2: ```suggestion
でも、ボクってばやさしいから、\n揺らすだけじゃなくて、\nキミがやる気になれるように「おまけ」してあげる❤\,But since I'm nice; I won't just bounce them for you. How about I give you a little "something" to get you extra motivated~❤
```

> ほら、おっぱいハグだぞ～❤\n揺れるおっぱいでもみくちゃにされちゃえ♪,Here you go! A big boobie hug~❤ Let yourself be smothered by my bouncing breasts!♪

Reviewer 2: Bouncing has been said a lot this interaction.
```suggestion
ほら、おっぱいハグだぞ～❤\n揺れるおっぱいでもみくちゃにされちゃえ♪,Here you go! A big boobie hug~❤ Smother yourself between my boobs as I jostle your face between them!♪
```

Author: 'Jostle' is kind of a big word for her. What about jiggle or shake?

Reviewer 2: Hmmm maybe "rub your face between them" then?

Author: Also he's not smothering himself, she's smothering him, so that change doesn't make sense.

顔をむぎゅって捕まえて❤\nうり、うり、うり～♪,I'll squeeze your face tight!❤ That's it. Come on. Just give in~♪
あひっ、あううっ、気持ちいいっ❤❤あああっ❤,Ahh; aghh; it feels so good!❤❤ Aaaah!❤
あへ、あへぇ・・・❤,Ah; aahhh...❤
おっとっと、気持ちよくしすぎて、射精させちゃった。\nこれ以上魔力を出しちゃうとカードも出せなくなっちゃうね。,Oops; I made it feel too good and accidently made you cum. If you lose any more magic power; you won't be able to summon cards anymore; will you?
ほら、命令♪僕のために、あいつらを倒してきて～❤\nいけいけ、やっちゃえー♪,Alright; it's an order!♪ Go defeat those guys for me.❤ Go on; beat them up~♪

> 冷静になりかけていた僕の脳は強烈な快楽により一瞬で塗り替えられてしまった。頭の芯がとろけたようになり、何も考えられない・・・。,Even though my brain had been on the verge of regaining composure; it was instantly overwhelmed by intense pleasure. My mind—my very self—seemed to melt; and I couldn't think of anything. Anything; but...

Reviewer 2: ```suggestion
冷静になりかけていた僕の脳は強烈な快楽により一瞬で塗り替えられてしまった。頭の芯がとろけたようになり、何も考えられない・・・。,Even though my brain had been on the verge of regaining composure; it was instantly overwhelmed by the intense pleasure. My mind—my very self—seemed to melt; and I couldn't think of anything. Anything; but...
```

その後僕は朦朧とした意識の中でカードを使い仲間たちを攻撃した。後ろでシャドーが嬉々としながら命令をする。その声に言われるがまま、僕はカードを使用していった。,After that; lost in my dazed state; I used my cards and attacked my own allies as Shadow continued to gleefully give me orders from behind.  I continued using my cards as she told me to; obediently hanging on every word.
魔力が足りない状態で無理にカードを使い、体中がきしむように痛む。しかしその痛みすらも、どこか遠いもののように感じる。それぐらい与えられた快感は強烈で、意識は雲の上にいるようにふわふわとしていた。,Forcing myself to use cards while so low on magical power made my whole body ache; but even that pain felt distant somehow. The pleasure I'd been given was so intense that my consciousness had been left floating dreamily; as if cradled by clouds.
一番ランクも高くサキュバスと戦ったこともある僕が、みんなを守らないといけないはずだった。彼女たちは、仲間だったはずの僕の突然の攻撃に驚き、そして・・・・。,I had the highest rank of all the cardmasters here; and had even fought succubi before. I should have been the one protecting everyone. The sudden betrayal of their ally and guardian had left them all stunned and helpless. And then...
あははは♪楽しかったねー♪\nみんなのびっくりした顔を見た？\n裏切られたときの人間の顔って最高♪,Ahahaha~♪ That was fun!♪ Did you see how surprised everyone was? Humans make the best faces when they're getting betrayed~♪

> でも別にいいよね？女の子はみんなサキュバスになっちゃうだけだし、\n男の子は気持ちよく吸い殺してもらえるし、\n誰も不幸になんてなってないもん❤,But it's okay; right? The girls will just become succubi; and the boys will be pleasurably drained to death; so it's not like anyone's really suffering~❤

Reviewer 2: "the boys will be pleasurably drained to death" is a little stiff. Try something that flows better in English rather than a direct translation:

```suggestion
でも別にいいよね？女の子はみんなサキュバスになっちゃうだけだし、\n男の子は気持ちよく吸い殺してもらえるし、\n誰も不幸になんてなってないもん❤,But it's okay; right? The girls will just become succubi; and the boys will experience their final moments in pure bliss; so it's not like anyone's really suffering~❤
```

Author: I'll take another stab at it, but your suggestion deletes all mention of being drained, which i think is pretty important

Author: How about 'and the boys will have all their energy drained in the nicest way'

Reviewer 2: Hmm I think "final moments" is nuanced enough. I think the initial suggestion is still better.

Author: I'll use it, but I think you're losing a big point of the sentence by cutting out 'drained.'

> くす❤さっきのぱふぱふの快感が抜けてきて、\n少し意識が晴れてきた？\nその泣きそうな顔。自分がしちゃったこと後悔してるのかな。,*Giggle*❤ Looks like the pleasure I gave you earlier is starting to wear off. Is your mind clearing up a bit? You're tearing up. Are you regretting what you did?

Reviewer 2: Hmm I don't think this translation is right, and it doesn't quite flow well either. A few things were missed here.

```suggestion
くす❤さっきのぱふぱふの快感が抜けてきて、\n少し意識が晴れてきた？\nその泣きそうな顔。自分がしちゃったこと後悔してるのかな。,*Giggle*❤ Looks like looks like that daze my Puff-Puff left you in is starting to fade. Feeling a little more clearheaded now? You look like you're about to cry! Are you regretting what you did?
```

> おっぱいのいいなりになって、人間を裏切っちゃったくせに、\nいまさらみっともないよ？ほらキミは、\nこのおっぱいのために、これまでのことぜーんぶ捨てちゃったの❤,You became a slave to my breasts and betrayed humanity; and you choose now to get this pathetic? You threw away everything you've been through for a nice pair of tits~❤

Reviewer 2: Hmm this sentence doesn't quite fit either. "and you choose now" implies the statement before was positive, but both parts of the sentence are insults so it doesn't really flow naturally.
```suggestion
おっぱいのいいなりになって、人間を裏切っちゃったくせに、\nいまさらみっともないよ？ほらキミは、\nこのおっぱいのために、これまでのことぜーんぶ捨てちゃったの❤,Despite the fact you were so willing to betray humanity just a moment ago—all because you can't say no to a pair of tits—now you decide to regret your actions? Everything you've done so far—you've thrown it all away for my boobs.❤
```

キミの足元の影を入り口にして、\n暗ーい世界へ、ずぶずぶ～♪,I'll use the shadow at your feet as an entrance; and sink you deep into darkness~♪
ここはボクの影の世界❤\n真っ暗でボク以外何も見えないでしょ？\nキミを今日からボクの兵隊として、この影の中で飼ってあげる❤,This is my shadow world.❤ It's pitch black in here. You can't see anything but me; can you? From today on; I'll keep you as my soldier; hidden in the dark~❤
嫌かな？嫌じゃないよね？\nだって、こんなにかわいいボクと一緒にいられるんだもん❤,You don't mind; do you? Of course you don't. After all; you get to be here with cute little me~❤
ほらぁ、まずは歓迎のあいさつ代わりに、\nキミのこと死ぬ寸前まで搾ってあげる❤,Now; as a way of saying 'welcome'; I'll milk you to the brink of death~❤
はひぃっ❤あうぅっ❤やわらかい・・・❤ああ・・・❤,Hahhh...❤ Aghh...❤ S-so soft...❤ Aah...❤
にひひ❤さっきまで後悔したような顔してたのに、\n快感ですぐ蕩けちゃったね❤,Nihihi~❤ You looked so guilty a minute ago; but your face melted instantly from pleasure.❤
暗ーい闇の中にいるとね、ちっぽけなことなんて、\n全部溶けてしまうの❤\n闇に飲み込まれて、気持ちよくまどろんじゃうの❤,When you're this deep in darkness; trivial things just dissolve away.❤ When they're swallowed by my shadows; everyone just drifts off; lost in bliss~❤

> キミもボクの影の中で、どろどろにとろけていいよ❤\nえいえい❤,I'll make sure you melt into goo lost in my shadow; too.❤ Here I go~❤

Reviewer 2: Hmm this first sentence doesn't quite flow well I think... What about this?

```suggestion
キミもボクの影の中で、どろどろにとろけていいよ❤\nえいえい❤,I'll make sure you melt into goo; so much you become part of my shadow~❤ Here I go~❤
```

Author: I can give it another go, but I dont think 'become part of my shadow' is right. It feels more like 'im going to do this to you WHILE you're lost in my shadow'

Reviewer 2: That works too, if you feel that makes more sense.

Author: That wasn't a suggestion, I was just saying that was what my original line was, compared to your making it about the MC becoming a part of her shadow.

Reviewer 2: So my main issue here is with the English itself rather than what is actually being said:
"goo lost in my shadow" doesn't make sense. Something is missing in the logic here...

Reviewer 2: Since we aren't landing on anything else. I'll go with my first suggestion, as I see no reason why that wouldn't fit.

> おちんちんズリズリされて気持ちいい？気持ちいいよね❤\nボクのサキュバスおっぱい❤みっちりおちんちんに絡みつくもんね❤,Does it feel good having your dick rubbed and stroked by my tits? It does; doesn't it?❤ My succubus breasts are the best~❤ The way they cling so tightly around your dick is incredible; isn't it?❤

Reviewer 2: ```suggestion
おちんちんズリズリされて気持ちいい？気持ちいいよね❤\nボクのサキュバスおっぱい❤みっちりおちんちんに絡みつくもんね❤,Does it feel good having your dick rubbed and stroked between my tits? It does; doesn't it?❤ My succubus breasts are the best~❤ The way they cling so tightly around your dick is incredible; isn't it?❤
```

> これからボクに命じられるまま、\nどんどんひどいことをして、どんどん堕ちていこうね❤,From now on; you'll do increasingly terrible things at my command; falling deeper and deeper into depravity~❤

Reviewer 2: increasingly terrible things feels a little weird to me as a phrase.

Author: Suggestion?

Reviewer 2: Sure thing. Hmm... What about this?
```suggestion
これからボクに命じられるまま、\nどんどんひどいことをして、どんどん堕ちていこうね❤,From now on; you must obey my commands. Falling deeper and deeper into depravity~❤
```

Author: That loses the part in the Japanese that says the things he does will keep getting worse

Reviewer 2: Hmm I left it out because I think the nuance is still there... Hmm... What about this then?
```suggestion
これからボクに命じられるまま、\nどんどんひどいことをして、どんどん堕ちていこうね❤,From now on; you must obey my commands. Every naughty order I give you making you fall deeper and deeper into depravity~❤
```

Author: I still feel it's lacking. It feels like she really wants to double up on the 'falling' idea here, so saying that things he's going to do are gonna get worse and worse is kinda important.

Reviewer 2: What about this?

```suggestion
これからボクに命じられるまま、\nどんどんひどいことをして、どんどん堕ちていこうね❤,From now on; you must obey my commands. Every order more naughty than the last; making you fall deeper and deeper into depravity~❤
```
